compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/irCodegenUtils.kt

```diff
@@ -165,6 +165,9 @@ fun IrDeclarationWithVisibility.getVisibilityAccessFlag(kind: OwnerKind? = null)
         ?: visibilityToAccessFlag[visibility]
         ?: throw IllegalStateException("$visibility is not a valid visibility in backend for ${ir2string(this)}")
 
+fun IrFunction.getVisibilityForDefaultArgumentStub(): Int =
```

Comment:
I'd implement it like this, and probably move it to `FunctionCodegen`:
```
when (visibility) {
    Visibilities.PUBLIC -> Opcodes.ACC_PUBLIC
    JavaVisibilities.PACKAGE_VISIBILITY -> AsmUtil.NO_FLAG_PACKAGE_PRIVATE
    else -> error("Default argument stub should be either public or package private: ${ir2string(this)}")
}
```

Reply:
I agree. Thanks. :)